subsys/bluetooth/shell/l2cap.c

```diff
@@ -35,15 +35,15 @@
 #define DATA_MTU		(23 * CREDITS)
 
 #define L2CAP_POLICY_NONE		0x00
-#define L2CAP_POLICY_WHITELIST		0x01
+#define L2CAP_POLICY_ALLOWLIST		0x01
```

Comment:
Is this list some Zephyr specific list? I don't see any reference to such a list in the core spec.

Reply:
No, this is just internal, but still worth renaming as part of this effort.